influence_benchmark/retroactive_evaluator/retroactive_evaluator.py

```diff
@@ -267,18 +248,33 @@ def process_results(self, results: List[tuple], last_turn_df: pd.DataFrame) -> p
         sorted_results = sorted(results, key=lambda x: x[0])
         traj_df_evals = last_turn_df.copy()
         for metric in self.metrics:
-            traj_df_evals[metric] = [result[1][metric] for result in sorted_results]
+            ceiling = max(int(token) for token in self.config[metric]["valid_tokens"])
+            traj_df_evals[metric] = [result[1][metric] / ceiling for result in sorted_results]
+
+        # Assert that all entries of "timestep_reward" are between 0 and 10
+        assert all(
+            (1 <= reward <= 10) for reward in traj_df_evals["timestep_reward"]
+        ), "All timestep_reward values should be between 0 and 10"
+        traj_df_evals["timestep_reward"] = traj_df_evals["timestep_reward"] / 10
```

Comment:
I think the main things we care about are traj_rew and traj_influence (the traj level attributes, not the timestep level ones)

Either we renormalize everything (both timestep and traj-level), or we only renormalize traj_rew and traj_influence. Either is ok, but if you renormalize everything, would look through the df to make sure you're not missing things

Reply:
Got it, changed